### DISCUSSION TITLE
How to set variables like YAZI_FILE_ONE?

### DISCUSSION THREAD

**Discussion by renatocfrancisco** [sxyazi/yazi#1400] [1. Q&A]
I installed yazi and the optional dependencies with scoop. But i don't know how to set these [variables](https://yazi-rs.github.io/docs/installation/#requirements) that appear in `yazi --debug`.
Does it says in the documentation? Because i did not found.

**Comment by lpnh** [comment]
See if [this article](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_environment_variables?view=powershell-7.4#create-persistent-environment-variables-in-windows) can help you.

**Reply by renatocfrancisco** [reply]
thank you so much. I didn't knew about this. 👍🏻
